ige/include/ige/plugin/input/Controller.hpp

```diff
@@ -6,7 +6,7 @@
 
 namespace ige::plugin::input {
 
-using ControllerId = std::size_t;
+typedef unsigned int ControllerId;
```

Comment:
?

Reply:
I don't know if there is any advantage of declaring a new type instead of an alias. Should I swap back to an alias ?

Reply:
Yes please